packages/ra-core/src/auth/useLogoutIfAccessDenied.spec.tsx

```diff
@@ -67,11 +79,12 @@ describe('useLogoutIfAccessDenied', () => {
     });
 
     it('should not logout if passed no error', async () => {
-        const { queryByText } = render(
+        const { queryByText } = renderComponent(
```

Comment:
We can either mock the `useHistory`, or provide a Router. Doing neither results in this: 
![image](https://user-images.githubusercontent.com/17087167/120549511-9cc7c200-c3eb-11eb-9f51-3f9876ee276b.png)

I chose to wrap them all in a router because

a) I prefer to avoid mocking if I don't need to
b) we'd have to wrap the new test in a router regardless, and so we'd be wrangling with both sets logic in the suite